modules/apps/object/object-rest-test/src/testFunctional/tests/ManageCategoriesForObjectEntries.testcase

```diff
@@ -36,9 +36,11 @@ definition {
 
 		ObjectAdmin.deleteAllCustomObjectsViaAPI();
 
-		TaxonomyVocabularyAPI.deleteTaxonomyVocabularyByErc(
-			externalReferenceCode = "erc",
-			groupName = "Guest");
+		for (var externalReferenceCode : list "erc,erc1") {
```

Comment:
Can we create a new macro to deleteAllTaxonomyVocabulary{} here? I'm not very sure if this can be implemented or not

Reply:
will done